packages/vue-app/template/head.js

```diff
@@ -0,0 +1,29 @@
+<%= isTest ? '/* eslint-disable quotes, semi, indent, comma-spacing, key-spacing, object-curly-spacing, space-before-function-paren  */' : '' %>
+const head = <%= serializeFunction(head) %>
+<%= isTest ? '/* eslint-enable quotes, semi, indent, comma-spacing, key-spacing, object-curly-spacing, space-before-function-paren */' : '' %>
+
+<%
+// Inject link with alias
+if (head.link) {
```

Comment:
This process is happening at runtime. We can prepare serialized data (as discussed in discord) for most efficient code